lib/injectable/modules/api_module.dart

```diff
@@ -2,10 +2,12 @@ import 'package:dio/dio.dart';
 import 'package:injectable/injectable.dart';
 import 'package:pretty_dio_logger/pretty_dio_logger.dart';
 
+import 'auth_interceptor.dart';
+
 @module
 abstract class ApiModule {
   @singleton
-  Dio get client => Dio()
+  Dio get client => Dio(BaseOptions(baseUrl: "http://192.168.8.151:4000/api/"))
```

Comment:
base url should be fetched from local env variables like you did with the login data on drinks app